client/src/document/ingredients/browser-compatibility-table/feature-row.tsx

```diff
@@ -276,14 +281,17 @@ function CellIcons({ support }: { support: bcd.SupportStatement | undefined }) {
   if (!supportItem) {
     return null;
   }
-  return (
+  return isOnlySupportedWithPrefix(support) ||
+    hasNoteworthyNotes(supportItem) ||
+    isOnlySupportedWithAltName(support) ||
+    supportItem.flags ? (
     <div className="bc-icons">
       {isOnlySupportedWithPrefix(support) && <Icon name="prefix" />}
       {hasNoteworthyNotes(supportItem) && <Icon name="footnote" />}
       {isOnlySupportedWithAltName(support) && <Icon name="altname" />}
       {supportItem.flags && <Icon name="disabled" />}
     </div>
-  );
+  ) : null;
```

Comment:
To avoid repetition:

```ts
const icons = [
      {isOnlySupportedWithPrefix(support) && <Icon name="prefix" />},
      {hasNoteworthyNotes(supportItem) && <Icon name="footnote" />},
      {isOnlySupportedWithAltName(support) && <Icon name="altname" />},
      {supportItem.flags && <Icon name="disabled" />},
].filter(Boolean);

return (icons.length ? `<div className="bc-icons">{icons}</div>` : null);
```

Reply:
```js
function CellIcons({ support }: { support: bcd.SupportStatement | undefined }) {
  const supportItem = getFirst(support);
  const icons = [
    isOnlySupportedWithPrefix(support) && <Icon name="prefix" />,
    supportItem && hasNoteworthyNotes(supportItem) && <Icon name="footnote" />,
    isOnlySupportedWithAltName(support) && <Icon name="altname" />,
    supportItem && supportItem.flags && <Icon name="disabled" />,
  ].filter(Boolean);

  return icons.length ? `<div className="bc-icons">{icons}</div>` : null;
}
```